modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,135 @@
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of the AWS Load Balancer Controller using AWS Load Balancer Operator
+
+After installing the Operator, you can create an instance of the `AWSLoadBalancerController` by using CLI. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The Operator reconciles only the resource with the name `cluster`.
+
+.Prerequisites
+
+* You have created the `echoserver` namespace.
+* You have an access to OpenShift CLI (oc).
+
+.Procedure
+
+. Create a `AWSLoadBalancerController` resource YAML file, for example, sample-aws-lb.yaml, as follows:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: Auto <3>
+  additionalResourceTags: <4>
+    example.org/cost-center: 5113232
+    example.org/security-scope: staging
+  ingressClass: cloud <5>
+  config:
+    replicas: 2 <6>
+  enabledAddons: <7>
+    - AWSWAFv2 <8>
+----
+
+<1> Defines the `AWSLoadBalancerControlller` resource.
+<2> Defines the `aws-load-balancer-controller` instance name. This instance name gets added as a suffix to all related resources.
+<3> Valid options are `Auto` and `Manual`. When the value is set to `Auto` the Operator attempts to determine the subnets that belong to the cluster and tags them appropriately. The Operator cannot determine the role correctly if the internal subnet tags are not present on internal subnet. If you installed your cluster on user-provided infrastructure, you can manually tag the subnets with the appropriate role tags and set the subnet tagging policy to `Manual`.
+<4> Defines the tags used by the controller when it provisions AWS resources.
+<5> The default value for this field is `alb`. The Operator provisions an `IngressClass` with the same name if it does not exist.
+<6> Specifies the number of replicas of the controller.
+<7> Specifies add-ons for AWS load balancers, which get specified through annotations.
+<8> Enables the `alb.ingress.kubernetes.io/wafv2-acl-arn` annotation.
+
+. Create a `AWSLoadBalancerController` resource:
++
+[source,terminal]
+----
+$ oc create -f sample-aws-lb.yaml
+----
+
+. After the AWS Load Balancer Controller is running, create a `deployment` resource:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echoserver
+  namespace: echoserver
+spec:
+  selector:
+    matchLabels:
+      app: echoserver
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: echoserver
+    spec:
+      containers:
+        - image: openshift/origin-node
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: echoserver
+          ports:
+            - containerPort: 8080
+----
+
+. Create a service:
```

Comment:
```suggestion
. Create a `service` resource:
```
Suggest adding a step creating it with the appropriate command

Reply:
Addressed